### DISCUSSION TITLE
douban  Request enhancement

### DISCUSSION THREAD

**Discussion by jzysl423** [Purfview/IMDb-Scout-Mod#105] [Ideas]
The mouse moves to the Douban icon, displays the movie information Douban

The image below shows the effect
![1](https://user-images.githubusercontent.com/92137562/168520583-f7f0a100-a0cb-49f0-ae44-ea9cc6cca002.jpg)
Script code
https://greasyfork.org/scripts/17663-douban-movie-info-for-rarbg

![2](https://user-images.githubusercontent.com/92137562/168520673-de5360dd-c0fc-4b63-b3ba-d434eeb68b49.jpg)

**Comment by Purfview** [comment]
I don't have time for this.

Ask  [tofu liang](https://greasyfork.org/en/users/11649-tofu-liang) to add such feature to IMDb Scout Mod.
It shouldn't be hard for him to port it here.
